pycaret/tests/test_time_series.py

```diff
@@ -470,6 +470,23 @@ def test_tune_model_custom_folds(load_data):
     assert len(metrics2) == custom_fold + 2  # + 2 for Mean and SD
 
 
+def test_tune_model_alternate_metric(load_data):
+    """tests model selection using non default metric"""
+    exp = TimeSeriesExperiment()
+    fh = 12
+    fold = 2
+
+    exp.setup(data=load_data, fold=fold, fh=fh, fold_strategy="sliding")
+
+    model_obj = exp.create_model("naive")
```

Comment:
is there a reason for testing the naive model? I image this makes test run faster as you only want to test non default parameters.

Reply:
Yes, it is to make it faster. I just want to test that the flow works with non-default parameters (so 1 model would be OK). There is already a flow that tests all models through tune_model with default arguments so I think this is a good compromise,